examples/canjs/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "napa": "~1.2.0",
+    "todomvc-common": "~0.3.0",
+    "jquery": "~2.1.3",
+    "todomvc-app-css": "~1.0.1"
+  },
+  "scripts": {
+    "postinstall": "napa https://github.com/bitovi/canjs.com/archive/v2.0.7.zip:canjs passy/canjs-localstorage#v0.2.0:canjs-localstorage"
```

Comment:
Oh ok.

Wouldn't it be easier to just open a PR there with a package.json? I'll make sure he merges fast.

// @passy